responsemanager/client.go

```diff
@@ -197,8 +199,8 @@ func (rm *ResponseManager) CancelResponse(p peer.ID, requestID graphsync.Request
 	}
 }
 
-// this is a test utility method to force all messages to get processed
-func (rm *ResponseManager) synchronize() {
+// Synchronize is a utility method that blocks until all current messages are processed
```

Comment:
part of the reason this is private is it really shouldn't be used outside of test

Reply:
this is not necessary with the new response-completion code you added so I've reverted it